<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Oblohka</class>
 <widget class="QMainWindow" name="Oblohka">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="obl1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>143</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="obl2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>143</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="obl3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>143</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushforobl1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Обложка 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushforobl2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Обложка 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushfoobl3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Обложка 3</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
